include/boost/range/irange.hpp

```diff
@@ -205,6 +205,22 @@ namespace boost
 
     template<typename Integer>
     integer_range<Integer>
+    irange(int last)
+    {
+        BOOST_ASSERT( 0 <= last );
+        return integer_range<Integer>(0, last);
+    }
```

Comment:
Instead of repeating the body of 2-arg method, this method can just call the 2-arg constructor with `first=0`.

```
template <typename Integer>
integer_range<Integer>
irange(Integer last) {
    return irange<Integer>(0, last)
}